src/components/learner-credit-management/BudgetDetailActivityTabContents.jsx

```diff
@@ -32,19 +33,22 @@ const BudgetDetailActivityTabContents = ({ enterpriseUUID, enterpriseFeatures })
     );
   }
 
+  const hasSpentTransactions = !!budgetActivityOverview.spentTransactions?.count > 0;
```

Comment:
nit: `!!` isn't really needed here.  Because of javascript operator precedence, this would be converting the count to a boolean, with the end result being `true > 0` (hich still works because javascript would then coerce that to `1 > 0`)  
```suggestion
  const hasSpentTransactions = budgetActivityOverview.spentTransactions?.count > 0;
```
Which would result in a `undefined > 0` check in the event there is no `spentTransactions.count`, but that's fine since `undefined` gets coerced to a `0` in arithmetic operations.